tests/coalaTest.py

```diff
@@ -51,35 +51,36 @@ def test_python_version_34(self):
 
     def test_did_nothing(self):
         retval, output = execute_coala(coala.main, 'coala', '-I',
-                                       '-S', 'default.enabled=false')
+                                       '-S', 'cli.enabled=false')
         self.assertEqual(retval, 2)
         self.assertIn('Did you forget to give the `--files`', output)
 
         retval, output = execute_coala(coala.main, 'coala', '-I',
                                        '-b', 'JavaTestBear', '-f', '*.java',
-                                       '-S', 'default.enabled=false')
+                                       '-S', 'cli.enabled=false')
         self.assertEqual(retval, 2)
         self.assertIn('Nothing to do.', output)
 
     def test_show_all_bears(self):
         with bear_test_module():
-            retval, output = execute_coala(coala.main, 'coala', '-B')
+            retval, output = execute_coala(
+                coala.main, 'coala', '-B', stdout_only=True)
```

Comment:
again I think `-I` should be used instead of silencing stderr.
stderr shouldnt normally have any output here, and this will be masking bugs in the future.